### PR TITLE
feat: added unit test coverage make targets

### PR DESCRIPTION
**Pull request by pawelpaszki**
## Description
feat: added unit test coverage make targets

## Verification Steps
1. Run `make test/output/coverage/report` without `coverage.out` file present in the root dir and confirm that `coverage.out file not found` was displayed
2. Run `make test/html/coverage/report` without `coverage.out` file present in the root dir and confirm that `coverage.out file not found` was displayed
3. Run `make test` and confirm that `cover.out` is generated and processed at the end of the test execution (certain files are removed from the coverage report (e.g. mocked files...))
4. Run `make test/output/coverage/report` with `coverage.out` file present in the root dir and confirm that coverage results are displayed in the terminal
5. Run `make test/html/coverage/report` with `coverage.out` file present in the root dir and confirm that coverage results are displayed in the browser

## Checklist (Definition of Done)

- [x] All acceptance criteria specified in JIRA have been completed
- ~~[ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)~~
- ~~[ ] Documentation added for the feature~~
- ~~[ ] CI and all relevant tests are passing~~
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- ~~[ ] Required metrics/dashboards/alerts have been added (or PR created).~~
- ~~[ ] Required Standard Operating Procedure (SOP) is added.~~
- ~~[ ] JIRA has created for changes required on the client side~~

### CODE REVIEW

**Comment by machi1990** [comment]
@VanillaSpoon ping.. want to have a look / review at this?

**Comment by VanillaSpoon** [comment]
Looks good, thanks for the clear verification steps. Nice to see the implementation too :)

File: pkg/workers/worker_interface.go [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 
 import (
-	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/metrics"
 	"sync"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/metrics"
 )
 
-//go:generate moq -out woker_interface_moq.go . Worker
+//go:generate moq -out worker_interface_moq.go . Worker
```
Comment (machi1990) [inline pkg/workers/worker_interface.go:9]: Good catch!

**Comment by pawelpaszki** [comment]
@machi1990 - can I get this PR approved given that it was verified?

**Review by valdar — APPROVED** [review]

**Review by machi1990 — APPROVED** [review]
lgtm

Thanks for the clear verification steps.

**Pull request merged**